src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -195,6 +202,13 @@ def execute(self, reqStatus):
             self.logger.info("%d requests information completely processed.", len(reqResults))
 
             for wflow in reqResults:
+                # perform site list updates
```

Comment:
I think this code has to be placed outside of this for loop (L197). Otherwise it will only get executed when there is other workflows in the queue for data placement (workflows sitting in `assigned`).